README.md

```diff
@@ -64,6 +64,15 @@ First you'll need a `config.ini` text file with values for `github_username` and
 See [instructions on how to get a GitHub OAuth token](https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line) (the token only needs the `public_repo` permission).
 This file is ignored in `.gitignore` for security reasons.
 
+For the MySQL database you should also add, the following values:
+```
+host = '127.0.0.1' #or whatever the IP address of your localhost is
```

Comment:
I think if you use `container_name` in the Docker compose file, you can also just specify that, which is a little easier.

I was playing with a similar setup the other day (initially copying what you did in your earlier PR!) and couldn't get the host to work so ended up using that approach instead.

https://github.com/ormestad/plates/pull/22/files